### PR TITLE
Added no stealth map creep fremen to d2k

### PR DESCRIPTION
**Pull request by reaperrr**
Needed for campaign (and possibly some multiplayer maps).

### CODE REVIEW

**Comment by penev92** [comment]
Can you please add it to the map importer?

Uncomment `////{ 641, Pair.new("", "Fremen") },// Fremen fremen non-stealth`, remove the explanation, and add the actor name.

**Comment by reaperrr** [comment]
Done.

**Comment by penev92** [comment]
Sorry I meant remove the `// Fremen fremen non-stealth` part.

File: mods/d2k/rules/infantry.yaml [hunk 1]
```diff
@@ -227,3 +227,22 @@ saboteur:
...
+		Name: Fremen
+		Description: Elite sniper infantry unit\n  Strong vs Infantry\n  Weak vs Vehicles
+	Buildable:
+		Queue: Infantry
+		BuildPaletteOrder: 105
+		Prerequisites: ~disabled
+	RevealsShroud:
+		Range: 4c768
+	Mobile:
+		Speed: 66	##33
```
Comment (abcdefg30) [inline mods/d2k/rules/infantry.yaml]: Why the comment here?

Comment (GraionDilach) [inline mods/d2k/rules/infantry.yaml]: I guess it's a remnant of Bolt's D2k PR.

Comment (reaperrr) [inline mods/d2k/rules/infantry.yaml]: Yes. He doubled the speed over the result of converting the original values to OpenRA.

Actor speed will be dealt with in a separate PR, so I didn't bother changing this as it's not a buildable actor anyway.

Comment (pchote) [inline mods/d2k/rules/infantry.yaml]: Maybe just set this to the current fremen speed (if it isn't already) and drop the comment to avoid controversy?

**Comment by reaperrr** [comment]
Updated.
- removed comment from importer
- made nsfremen inherit speed from buildable fremen

File: mods/d2k/rules/infantry.yaml [hunk 2]
```diff
@@ -227,3 +227,20 @@ saboteur:
...
 
+nsfremen:
+	Inherits: fremen
+	Valued:
+		Cost: 0
+	Tooltip:
+		Name: Fremen
+		Description: Elite sniper infantry unit\n  Strong vs Infantry\n  Weak vs Vehicles
+	Buildable:
+		Queue: Infantry
```
Comment (pchote) [inline mods/d2k/rules/infantry.yaml]: These will be inherited, so you only really need the prereqs.

File: mods/d2k/rules/infantry.yaml [hunk 3]
```diff
@@ -227,3 +227,20 @@ saboteur:
 	Voiced:
 		VoiceSet: SaboteurVoice
 
+nsfremen:
+	Inherits: fremen
+	Valued:
+		Cost: 0
+	Tooltip:
+		Name: Fremen
```
Comment (pchote) [inline mods/d2k/rules/infantry.yaml]: This will be inherited too.

File: mods/d2k/rules/infantry.yaml [hunk 4]
```diff
@@ -227,3 +227,20 @@ saboteur:
...
+		Description: Elite sniper infantry unit\n  Strong vs Infantry\n  Weak vs Vehicles
+	Buildable:
+		Queue: Infantry
+		BuildPaletteOrder: 105
+		Prerequisites: ~disabled
+	RevealsShroud:
+		Range: 4c768
```
Comment (pchote) [inline mods/d2k/rules/infantry.yaml]: Maybe inherit this?

**Comment by reaperrr** [comment]
Updated.

File: mods/d2k/rules/infantry.yaml [hunk 5]
```diff
@@ -227,3 +227,16 @@ saboteur:
 	Voiced:
 		VoiceSet: SaboteurVoice
 
+nsfremen:
+	Inherits: fremen
+	Valued:
+		Cost: 0
```
Comment (pchote) [inline mods/d2k/rules/infantry.yaml]: This is used by a lot more than just production, so it might be safer to keep this inherited as well?

Comment (reaperrr) [inline mods/d2k/rules/infantry.yaml]: Good point.

**Comment by reaperrr** [comment]
Updated.

**Comment by abcdefg30** [comment]
Looks good to me. :+1: / :white_check_mark:

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/01097bc17f82b5d5a296374bf935281a071d317b...b007210631c2a1f149a7e5278ca8ff94c1894fd1)
